### PR TITLE
tools:add fpgainfo DFL driver support

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/libboard/board_common/board_common.c [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
-#endif // HAVE_CONFIG_H
+#endif // HAVE_CONFIG_Hrm
```
Comment (tswhison) [inline tools/libboard/board_common/board_common.c:29]: Remove the "rm" from the end of "HAVE_CONFIG_H"

File: tools/libboard/board_common/board_common.c [hunk 2]
```diff
@@ -44,6 +44,17 @@
 
 #include "board_common.h"
 
+#define SYSFS_PATH_MAX 256
```
Comment (tswhison) [inline tools/libboard/board_common/board_common.c:47]: remove

File: tools/libboard/board_common/board_common.c (hunk 1 above)
Comment (anandaravuri) [reply tools/libboard/board_common/board_common.c]: fixed remove rm

File: tools/libboard/board_common/board_common.c (hunk 2 above)
Comment (anandaravuri) [reply tools/libboard/board_common/board_common.c]: fixed removed  SYSFS_PATH_MAX

File: tests/board/test_board_n3000.cpp [hunk 3]
```diff
@@ -214,27 +219,40 @@ TEST_P(board_vc_c_p, board_vc_2) {
...
-	EXPECT_EQ(parse_fw_ver(NULL, fw_ver, SYSFS_MAX_SIZE), FPGA_INVALID_PARAM);
-
+/**
+* @test       board_n3000_3
+* @brief      Tests: parse_fw_ver
+* @details    Validates parse fw version  <br>
+*/
+TEST_P(board_dfl_n3000_c_p, board_n3000_3) {
+
+	char buf[SYSFS_MAX_SIZE];
```
Comment (tswhison) [inline tests/board/test_board_n3000.cpp:229]: SYSFS_PATH_MAX

**Pull request merged**